apps/E2E/src/ButtonLegacy/specs/ButtonLegacy.spec.win.ts

```diff
@@ -27,22 +26,39 @@ describe('Button Legacy Accessibility Testing', () => {
     await ButtonLegacyPageObject.scrollToTestElement();
   });
 
-  it('Button - Validate accessibilityRole is correct', async () => {
-    await expect(await ButtonLegacyPageObject.getAccessibilityRole()).toEqual(BUTTON_A11Y_ROLE);
+  it('Validate "accessibilityRole" defaults to Button "ControlType" element attribute.', async () => {
```

Comment:
Let's remove the "ControlType" part

'Validate "accessibilityRole" defaults to ControlType.Button'